src/Uno.UI/UI/Xaml/IFrameworkElementImplementation.Android.tt

```diff
@@ -76,12 +76,12 @@ namespace <#= mixin.NamespaceName #>
 
 		public double ActualWidth
 		{
-			get { return _actualSize.Width; }
+			get { return _currentSize.Width; }
```

Comment:
these kinds of renamings just create noise

Reply:
Yep but "actual size" was a misleading term here. It's a [french-english false friend](http://www.orbilat.com/Languages/French/Vocabulary/French-English-False_friends.html)

Reply:
No... it's the backing value for `ActualWidth` and `ActualHeight`, therefore it's called `actualSize`.